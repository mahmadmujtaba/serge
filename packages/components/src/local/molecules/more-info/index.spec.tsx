/* global it expect */

import React from 'react'
import renderer from 'react-test-renderer'
import MoreInfo from './index'

it('MoreInfo renders correctly', () => {
  const tree = renderer
    .create(<MoreInfo description="The percentage fuel remaining">Fuel</MoreInfo>)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
