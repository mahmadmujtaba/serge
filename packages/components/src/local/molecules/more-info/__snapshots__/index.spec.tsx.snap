// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoreInfo renders correctly 1`] = `
<div
  className="childnode"
>
  Fuel
  <div
    className="icon"
    onClick={[Function]}
  >
    <svg
      aria-hidden={true}
      className="MuiSvgIcon-root"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-6h2v6zm0-8h-2V7h2v2z"
      />
    </svg>
  </div>
</div>
`;
